package-lock.json

```diff
@@ -26,8 +26,8 @@
       "dev": true,
       "requires": {
         "@angular/tsc-wrapped": "4.1.3",
-        "minimist": "^1.2.0",
```

Comment:
Why is this caret symbol removed from versioning ? Yes I understand it is generated file do we have know the reason for this ? This applies to all changes in this file.

Reply:
Not sure.. It was changed automatically on npm install